mscore/editstafftype.cpp

```diff
@@ -485,12 +485,14 @@ void EditStaffType::tabStemThroughCompatibility(bool checked)
 void EditStaffType::updatePreview()
       {
       setFromDlg();
-      if (preview) {
+      if (preview && preview->score()) {
```

Comment:
There should be no runtime check for preview or preview->score(). Both are guaranteed.  Also the check in EditStaffType() - readScore() makes no sense as the score should always load.
Runtime checks for program errors are not acceptable in release code. Use assert() to document a guaranteed condition.